### PR TITLE
More explicit empty profile message

### PR DESCRIPTION
**Pull request by Pytal**
@nextcloud/designers feel free to adjust :)

| Before | After |
| ---       | ---    |
|  |   |

### CODE REVIEW

**Comment by Pytal** [comment]
@nextcloud/designers changes welcome :)

File: core/src/views/Profile.vue [hunk 1]
```diff
@@ -142,7 +142,7 @@
 							fill-color="var(--color-text-maxcontrast)"
 							:size="60" />
 						<h3>{{ emptyProfileMessage }}</h3>
-						<p>{{ t('core', 'The headline and about sections will show up here') }}</p>
+						<p>{{ t('core', 'Your added info will show up here') }}</p>
```
Comment (nimishavijay) [inline core/src/views/Profile.vue:145]: Does this message have to be different depending on if you are on your own profile or someone else's profile?

**Comment by skjnldsv** [comment]
Still relevant @Pytal :)

**Comment by skjnldsv** [comment]
Closing this pull request due to lack of recent activity and updates. We appreciate your contribution and encourage you to reopen or provide further updates if necessary. :relaxed: 
Our aim is to keep the project moving forward with active collaboration. Thank you for your understanding and continued support! :pray:

**Pull request closed without merging**